### PR TITLE
Document the convox.agent label

### PR DESCRIPTION
**Pull request by ianmalott**
Documentation for https://github.com/convox/rack/pull/1986

## Release Playbook
- [x] Rebase against master
- [x] Code review
- [x] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _reference/docker-compose-labels.md [hunk 1]
```diff
@@ -21,6 +22,15 @@ labels:
   - <a href="#convoxstart">convox.start.shift</a>
 </pre>
 
+## convox.agent
+
+The `convox.agent` label allows you to run a service as an agent. In other words, labeling a service as an agent tells the platform to schedule one process of the service per instance in your cluster.
```
Comment (ddollar) [inline _reference/docker-compose-labels.md]: Can probably tighten this up into one sentence.   The `convox.agent` label allows you to run one process...

File: _reference/docker-compose-labels.md [hunk 2]
```diff
@@ -21,6 +22,15 @@ labels:
...
+
+    labels:
+      - convox.agent=true
+
+For an example of an app you would run as an agent, see our example [DataDog Agent App](https://github.com/convox-examples/dd-agent).
```
Comment (ddollar) [inline _reference/docker-compose-labels.md]: Second "example" feels redundant.. could say something like   See [convox-examples/datadog-agent](url) for an example of...

File: _reference/docker-compose-labels.md (hunk 1 above)
Comment (ddollar) [inline _reference/docker-compose-labels.md]: Might also want to give a brief description of why/when they might want to use this label.

Comment (ianmalott) [inline _reference/docker-compose-labels.md]: Yep, good calls. Thanks.

File: _reference/docker-compose-labels.md (hunk 2 above)
Comment (ianmalott) [inline _reference/docker-compose-labels.md]: Agreed, thanks. Fixed.

**Pull request merged**